src/main/java/module/ifa/PluginIfaUtils.java

```diff
@@ -237,7 +237,7 @@ private static void insertMatches(Date from, Date to) throws Exception {
 		Date start = DateTimeUtils.getDateWithMinTime(from);
```

Comment:
This section uses java Dates and Calendar rather than HODateTime, should it all be HODateTime for consistency?

Reply:
Done: 3b99f83a9143572e21d4b1e4d7981c832d247f39 refactors the IFA module